test/new-e2e/scenarios/system-probe/main.go

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	systemProbe "github.com/DataDog/datadog-agent/test/new-e2e/system-probe"
+)
+
+func run(envName, securityGroups, subnets, x86InstanceType, armInstanceType string, destroy bool) error {
+	systemProbeEnv, err := systemProbe.NewTestEnv(envName, securityGroups, subnets, x86InstanceType, armInstanceType)
+	if err != nil {
+		return err
+	}
+
+	if destroy {
```

Comment:
If `destroy` is true, but the environment doesn't exist, will it create it and then immediately destroy it?

Reply:
Yes

Reply:
Can we fix that? That seems undesirable.

Reply:
This is currently not possible with the API the e2e-testing-squad has developed. 
I have made a PR to allow this: https://github.com/DataDog/datadog-agent/pull/15621